### PR TITLE
Preserve data :green_circle: 

### PR DESCRIPTION
**Pull request by HAMMAS-SALEEM**
### In this branch, We (@HAMMAS-SALEEM, @Lichapa ) worked on the following things :+1: 

:heavy_check_mark: Preserved Data in JSON files
:heavy_check_mark: Followed GItflow
:heavy_check_mark: Followed Ruby best practices
:heavy_check_mark: Followed Linters
:heavy_check_mark: Documented work in a professional way

### CODE REVIEW

File: modules/preserve_data.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# require './student'
+# require './teacher'
+# require './rental'
```
Comment (Meltrust) [inline modules/preserve_data.rb]: - [ ] Please, kindly remove this old piece of commented code, in order to preserve the cleanliness of the repo.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @HAMMAS-SALEEM

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice read-write implementation.✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/preserve_data.rb (hunk 1 above)
Comment (HAMMAS-SALEEM) [reply modules/preserve_data.rb]: OK Thanks, I would implement the changes

**Review by Meltrust — APPROVED** [review]
Hi @HAMMAS-SALEEM,


## Wow, you did it 🎉
Brilliant
## Thank you for deleting the commented code, as requested in the previous review💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**